src/main/java/com/butbetter/storage/model/ProductInformation.java

```diff
@@ -0,0 +1,100 @@
+package com.butbetter.storage.model;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+public class ProductInformation {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name="id", insertable = false, updatable = false, nullable = false)
```

Comment:
I already marked that @Nullable is better performing in the address class. 
Please remove nullable = false here too and replace it with the @NotNull annotation, if it is not a primitive
datatype

Reply:
Is same as before NotNull is annotation from beanvalidation and Column is annotation from Hibernate.
And better performing is not really true.